src/config/constants.js

```diff
@@ -32,6 +32,10 @@ const constants = {
           account
         })
     }
+  },
+  DISCORD: {
+    url: 'https://discord.gg/MRjKBXtaDt',
+    target: 'Discord StarkGate Channel'
```

Comment:
Why do we need `target`?

Reply:
when specifying a target- if a user opens the link more than once, on the second time it'll focus on the previously opened tab (not open a new one).